### PR TITLE
Added a 'strings-no-comments-swift3' template. Fixes #222

### LINKED ISSUE #222: Proposal: remove default translation comments from strings template
@AliSoftware @MrAlek 

I'd like to propose something that is the total opposite of https://github.com/AliSoftware/SwiftGen/issues/58: **removing** doc comments from the strings template.

My main issue with it is that every time I modify the translation of a key, it modifies the generated strings enum. This clutters the diff and my app needs to recompile – which is a practical issue with swift as the compiler is rather slow.

The argument for *adding* translation comments in https://github.com/AliSoftware/SwiftGen/issues/58 was it was nice to command+click and see the actual default translation. I fail to see how that is useful – the key should be descriptive enough that you'd know what the string is meant for. I personally never search for a key to find out its translation because frankly it shouldn't matter what it translates to, what matter is what it describes, and that's the role of the key.

I ended up creating a template that is the exact same one that swiftgen provides, minus the comment, which prevents me from getting all the new template updates. What I would propose is to either remove the comment altogether, or at the very least make it opt-in.

How does that sound? Thanks!

### PR DESCRIPTION
**Pull request by ldiqual**
@AliSoftware As per your comment in https://github.com/AliSoftware/SwiftGen/issues/222#issuecomment-264660153

Fixes #222.

 - Added a `strings-no-comments-swift3` template
 - Added a unit test to make sure the output is correct. Output was generated this way:

```
./build/swiftgen/bin/swiftgen strings -t no-comments-swift3 ./UnitTests/fixtures/Localizable.strings -o ./UnitTests/expected/Strings-File-NoComments-Swift3.swift.out
```

 - Added a line in the CHANGELOG

Let me know if you have any question. thanks!

### CODE REVIEW

**Review by AliSoftware — APPROVED** [review]

**Comment by AliSoftware** [comment]
Looks good to me from GitHub at least.
Will merge after the 4.0.1 patch release (as adding a new template is a minor version, not just patch, and will go in upcoming 4.1.0)

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +28,9 @@ _None_
 * Strings dot-syntax template: use `enum`s for namespacing instead of `struct`s.  
   [David Jennes](https://github.com/djbe)
   [#218](https://github.com/AliSoftware/SwiftGen/pull/218)
+* Added a `strings-no-comments-swift3` template that does not include the default translation of each key.  
+  [Loïs Di Qual](https://github.com/ldiqual)
+  [#222](https://github.com/AliSoftware/SwiftGen/issues/222)
```
Comment (AliSoftware) [inline CHANGELOG.md]: I've just released 4.0.1 (while your PR is not part of it) and then merged master into your branch to update your PR… but now your CHANGELOG entry seems to have been merged in the wrong CHANGELOG section :wink: whoops.

Could you please move it to the new `### New Features` subsection **of the `## master` section**?

Thx!

**Review by AliSoftware — CHANGES_REQUESTED** [review]

**Comment by ldiqual** [comment]
@AliSoftware Done.

**Review by AliSoftware — APPROVED** [review]
:+1:

**Pull request merged**